lib/livebook/notebook/explore/elixir_and_livebook.livemd

```diff
@@ -37,28 +37,6 @@ return the Elixir version.
 Note you can also press <kbd>tab</kbd> to cycle across the completion
 alternatives.
 
-## Imports
-
-You can import Elixir modules to make the imported functions visible
-to all subsequent cells. Usually you want to keep `import`, `alias`, and
-`require` in the first section, as part of the notebook setup.
-
-For instance, you can import `IEx.Helpers` and bring all of the amazing
-conveniences in Elixir's shell to your notebook:
-
-```elixir
-import IEx.Helpers
-```
-
-```elixir
-h(Enum.map())
-```
-
-```elixir
-# Sidenote: http://www.numbat.org.au/thenumbat
-i("I ❤️ Numbats")
-```
-
```

Comment:
I am thinking this section is no longer very much relevant. It is more about Elixir than Livebook too.

Reply:
We do have more intellisense now, but I like how we show the use of `IEx.Helpers` in particular, so it's clear that people can call stuff like `IEx.Helpers.flush()` and it works. Not sure though ^^